### DISCUSSION TITLE
Pause/Resume Print

### DISCUSSION THREAD

**Discussion by HalyconExport** [mriscoc/Ender3V2S1#467] [Q&A] [answered]
Good Morning Everyone. I have had this issue since installing, it is when I pause a print (either by the runout sensor or manually) once it resumes it will be printing as if it thinks it is shifted left a few cm so the print beyond the resume does not match with the bottom portion, and is shifted over.
![PauseResume](https://user-images.githubusercontent.com/91285828/198334020-b12d74ca-7bc2-488c-aa74-7381f803b572.jpg)

**Comment by mriscoc** [comment, marked as answer]
The printer is trying to go off physical limit when the head is parked. Verify your physical setttings https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Reply by HalyconExport** [reply]
Thanks!

**Reply by HalyconExport** [reply]
Just got a chance to fix it. It was 232 on both x and y max values with the sprite pro installed. Thank you so much!
